docs/Running-behind-nginx.md

```diff
@@ -1,8 +1,42 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx #
 
-To pass Netdata via a nginx, use this:
+## Intro ##
 
-### As a virtual host
+[Nginx](https://nginx.org/en/) is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server used to host websites and applications of all sizes. 
+
+The software is known for its low impact on memory resources, high scalability, and its modular, event-driven architecture which can offer secure, predictable performance.
+
+## Why Nginx ##
+
+Nginx is used and useful in cases when you do not want to access Netdata via the default `IP ADDRESS:19999`.
+
+With Nginx, the data is accessed via human-readable domains such as `example.com` or `sub.example.com`. 
```

Comment:
I don't need nginx for that, I just add a DNS entry. I do need nginx if I want to access different instances of netdata by using 'host/netdata/server1', 'host/netdata/server2'

Reply:
Also, in versions prior to 1.16.0, using a proxy was the only available way to encrypt the communication (https).

Reply:
Sure, you do need Nginx if the DNS entry is to work. 
A lot of cases, it is easier to remember the domains than IPs. 

More than to just add a DNS entry, you need to configure Nginx to serve the domain.

Reply:
No, I mean that I can just have a DNS entry pointing to the IP of my netdata! Put the following in your `/etc/hosts` and go to http://netdata.mydomain.com:19999. You will see netdata there.
```
127.0.1.1	netdata.mydomain.com
```

Reply:
Also, you didn't add the two valid reasons I mentioned for using a reverse proxy:
- Password-protect access to netdata, until distributed authentication is implemented via the netdata cloud Sign In mechanism.
- A proxy was necessary to encrypt the communication to netdata, until v1.16.0, which provided TLS (https) support.

Reply:
I have removed the previous statement 

> Nginx is used and useful in cases when you do not want to access Netdata via the default `IP 
ADDRESS:19999`.

>With Nginx, the data is accessed via human-readable domains such as `example.com` or >`sub.example.com`

and added the valid reasons.